scripts/release/release.js

```diff
@@ -0,0 +1,363 @@
+/******************************************************************************
+ * Scaleway CLI release script.
+ *
+ * This script will trigger a release process for the scaleway Go SDK.
+ *
+ * The script will proceed as follow:
```

Comment:
what about building docker images, and uploading them ? 

> docker buildx build --platform=linux/amd64,linux/arm/7,linux/arm64,linux/i386 -t scaleway/scaleway-cli:yourawesomev2tag . --push

Will this be added later ?  Do we have an issue to track it ?